easybuild/easyconfigs/n/NGSpeciesID/NGSpeciesID-0.1.2.1-foss-2021b.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'PythonPackage'
+
+name = 'NGSpeciesID'
+version = '0.1.2.1'
+
+homepage = 'https://github.com/ksahlin/NGSpeciesID'
+description = "NGSpeciesID is a tool for clustering and consensus forming of targeted ONT reads."
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['e7c94b147ceff184ca964a7adbcc0540dac7303136ebc53033245f8e33bd7faa']
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('python-parasail', '1.2.4'),
+    ('edlib', '1.3.9'),
+    ('medaka', '1.6.0'),
+    ('spoa', '4.0.7'),
+    ('Racon', '1.5.0'),
+    ('minimap2', '2.22'),
+]
+
+download_dep_fail = True
+use_pip = True
+
+# strip away too strict version requirement for parasail and 
```

Comment:
Bit nitpicky, but is there a typo in the comment here?

Reply:
The stripping isn't needed at all actually for this version, so I've removed it in d7478421df